test/colon/element-test.js

```diff
@@ -1287,4 +1287,32 @@ testHelpers.makeTests("can-stache-bindings - colon - element", function(name, do
 			console.groupCollapsed = groupCollapsed;
 		}
 	});
+
+	testIfRealDocument("Bracket Expression with :to bindings", function () {
+		var demoContext = new DefineMap({
+			person: {
+				name: 'Matt'
+			}
+		});
+
+		var SourceComponentVM = DefineMap.extend("SourceComponentVM", {
+			name: {
+				value: 'Kevin'
```

Comment:
Can you use `default` instead of `value`?

Reply:
Sure was just trying to be consistent with what i'd seen in other tests. np.